Libraries/UtilLib/src/test/java/com/artardev/util/numeric/NumberUtilFibonacciTest.java

```diff
@@ -0,0 +1,32 @@
+package com.artardev.util.numeric;
+
+import com.artardev.util.numeric.data.IntIntDataInfo;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Collection;
+import java.util.List;
+
+@RunWith(Parameterized.class)
+public class NumberUtilFibonacciTest {
+    IntIntDataInfo dataInfo;
+    public NumberUtilFibonacciTest(IntIntDataInfo dataInfo) {
+        this.dataInfo = dataInfo;
+    }
+    @Parameterized.Parameters
+    public static Collection<IntIntDataInfo> createData()
+    {
+        return List.of( new IntIntDataInfo(1, 0),
+                        new IntIntDataInfo(2, 1),
+                        new IntIntDataInfo(3, 1),
+                        new IntIntDataInfo(4, 2),
+                        new IntIntDataInfo(5, 3));
+    }
+    @Test
+    public void givenValue_WhenIndex_ThenReturnsNumber()
+    {
+        Assert.assertEquals(dataInfo.expected, NumberUtil.fibonacciNumber(dataInfo.input));
+    }
+}
```

Comment:
These test cases look good. Could you show me how you run these unit tests? This might help me set up workflows for this repo

Reply:
Yes please. If possible could you write the steps to replicate it in the README.md file it would be great. Alongside that the prerequisites that would be necessary for running the repository.